samples/snippets/src/test/java/com/google/cloud/examples/securitycenter/snippets/BigQueryExportIT.java

```diff
@@ -0,0 +1,151 @@
+  public static void requireEnvVar(String envVarName) {
+    assertWithMessage(String.format("Missing environment variable '%s' ", envVarName))
+        .that(System.getenv(envVarName))
+        .isNotEmpty();
+  }
+
+  @BeforeClass
+  public static void setUp() throws IOException {
+    final PrintStream out = System.out;
+    stdOut = new ByteArrayOutputStream();
+    System.setOut(new PrintStream(stdOut));
+
+    requireEnvVar("GOOGLE_APPLICATION_CREDENTIALS");
+    requireEnvVar("SCC_PROJECT_ID");
+
+    // Create a BigQuery dataset.
+    createBigQueryDataset(BQ_DATASET_NAME);
+    // Create export request.
+    String filter = "severity=\"LOW\" OR severity=\"MEDIUM\"";
+    CreateBigQueryExport.createBigQueryExport(
+        String.format("projects/%s", PROJECT_ID), filter, BQ_DATASET_NAME, BQ_EXPORT_ID);
+
+    stdOut = null;
+    System.setOut(out);
+  }
+
+  @AfterClass
+  public static void cleanUp() throws IOException {
+    final PrintStream out = System.out;
+    stdOut = new ByteArrayOutputStream();
+    System.setOut(new PrintStream(stdOut));
+
+    // Delete BigQuery Dataset and export request.
+    deleteBigQueryDataset(BQ_DATASET_NAME);
+    DeleteBigQueryExport.deleteBigQueryExport(
+        String.format("projects/%s", PROJECT_ID), BQ_EXPORT_ID);
+
+    stdOut = null;
+    System.setOut(out);
+  }
+
+  private static void createBigQueryDataset(String datasetName) {
+    try {
+      BigQuery bigquery = BigQueryOptions.getDefaultInstance().getService();
+
+      DatasetInfo datasetInfo = DatasetInfo.newBuilder(datasetName).build();
+
+      Dataset newDataset = bigquery.create(datasetInfo);
+      String newDatasetName = newDataset.getDatasetId().getDataset();
+      System.out.println(newDatasetName + " created successfully");
+    } catch (BigQueryException e) {
+      Assert.fail("Dataset was not created. \n" + e);
+    }
+  }
+
+  private static void deleteBigQueryDataset(String datasetName) {
+    try {
+      BigQuery bigquery = BigQueryOptions.getDefaultInstance().getService();
+      Assert.assertTrue("Deleted BigQuery dataset", bigquery.delete(datasetName));
+    } catch (BigQueryException e) {
+      Assert.fail("Dataset was not deleted. \n" + e);
+    }
+  }
+
+  @Before
+  public void beforeEach() {
+    stdOut = new ByteArrayOutputStream();
+    System.setOut(new PrintStream(stdOut));
+  }
+
+  @After
+  public void afterEach() {
+    stdOut = null;
+    System.setOut(null);
+  }
+
+  @Test
+  public void testGetBigQueryExport() throws IOException {
+    GetBigQueryExport.getBigQueryExport(String.format("projects/%s", PROJECT_ID), BQ_EXPORT_ID);
+    assertThat(stdOut.toString()).contains(BQ_EXPORT_ID);
+  }
+
+  @Test
+  public void testListBigQueryExports() throws IOException {
+    ListBigQueryExports.listBigQueryExports(String.format("projects/%s", PROJECT_ID));
+    assertThat(stdOut.toString()).contains(BQ_EXPORT_ID);
+  }
+
+  @Test
+  public void testUpdateBigQueryExport() throws IOException {
```

Comment:
Should we add test cases for `create` and `delete` as well?

Reply:
Hi Shabir, the create and delete are being tested as part of the setup and cleanup methods.
I've included an assert function in the cleanup to make sure that we are not missing any exceptions.